src/ExceptionConverter.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\FtpExtractor;
+
+use Keboola\Component\UserException;
+use League\Flysystem\FileNotFoundException;
+
+final class ExceptionConverter
+{
+    public static function resolve(\Throwable $e): void
+    {
+        if (self::isUserException($e)) {
```

Comment:
Tady bych zkusil nějak líp pořešit ty konkrétní výjimky (https://github.com/keboola/ex-ftp/pull/20#discussion_r299956844). Jakože - zkontrolovat, že je to fakt ta výjimka kterou očekáváme.

Reply:
Výjimka se tam vyhazuje na 4 místech v tom SftpAdapteru tak bych to klidně matchoval podle substringu té message třeba.